jcl/jpp_configuration.xml

```diff
@@ -470,4 +470,18 @@
 		<parameter name="jxerules:outputdir" value="java/lang"/>
 	</configuration>
 
+	<configuration
+		  label="SIDECAR18-TOOLS-OPENJ9"
+		  outputpath="SIDECAR18-TOOLS/src"
+		  dependencies="SIDECAR18-SE-OPENJ9"
+		  jdkcompliance="1.8">
+		<classpathentry kind="src" path="/pConfig SIDECAR18-SE-OPENJ9"/>
+		<classpathentry kind="lib" path="/binaries/vm/third/rt-compressed.sun180.jar"/>
+		<classpathentry kind="lib" path="/binaries/common/ibm/ibmjzos.jar"/>
+		<source path="src/openj9.dtfj/share/classes"/>
+		<source path="src/openj9.dtfjview/share/classes"/>
+		<source path="src/openj9.traceformat/share/classes"/>
+		<parameter name="macro:define" value="com.ibm.oti.vm.library.version=29"/>
```

Comment:
Does this need a `msg:outputdir` to match other configs?

Reply:
I removed it because setting "com/ibm/oti/util" is wrong (its also wrong in other non-base configs). The code controlled by this config doesn't use messages, and if it did a new message handler would need to be written to read from a different location. Messages for this tools code should not go into the same messages file used for bootstrap classes.  I'm not sure what happens if you try to use an external message without a `msg:outputdir` set, ideally an error should occur, but I'll have to try it.

Reply:
Just looked at `com/ibm/jpp/om/ExternalMessagesExtension.java` which has following code snippet for external message file directory:
````
return new File(new File(builder.getOutputDir(), messagesPath), messagesFile);
````
It appears `com/ibm/oti/util` is a relative path to the builder output directory. If it is changed to something like `com/ibm/oti/util-temp`, I can see `ExternalMessages.properties` is generated in this new folder. Removing `msg:outputdir` appears no effect because of the default value set by `private String messagesPath = "com/ibm/oti/util";`.
Agreed that this tool pconfig doesn't use external messages (just like when they live in a different repo before).

Reply:
How about if I set `<parameter name="msg:outputdir" value=""/>` since using `com/ibm/oti/util` conflicts with bootstrap classes / java.base, and this code doesn't use external messages anyway.

Reply:
Sounds good.

Reply:
Updated